pyatv/interface.py

```diff
@@ -1249,6 +1250,45 @@ async def text_set(self, text: str) -> None:
         raise exceptions.NotSupportedError()
 
 
+class TouchGestures(ABC):
+    """Base class for touch gestures."""
+
+    @feature(63, "TouchSwipe", "Touch swipe from given coordinates and duration.")
+    async def touch_swipe(
+        self, start_x: int, start_y: int, end_x: int, end_y: int, duration_ms: int
```

Comment:
I believe this is fine for simpler methods like this. My suggestion would be to add another method, accepting a sequence of something like "swipe"-objects (basically a dataclass with all arguments to this method as members). That would allow for definition of swipe patterns, something like this:

```python
SWIPE_RIGHT_LEFT = [
    Swipe(0, 0, 500, 0, 100),  # X: 0 -> 500 (right)
    Swipe(500, 0, 0, 0, 100),  # X: 500 -> 0 (left)
]

atv.touch.swipe_sequence(SWIPE_RIGHT_LEFT)
```

I'm up for discussion here regarding names and how to design this/these method(s), but I hope you get what I mean. I still see value in providing a very simple to use method like current version of `touch_swipe`, but allow for more advanced interactions as well. I would also be open to ship some default swipe actions in pyatv directly for common patterns.